### PR TITLE
Added TaskWarrior module

### PR DESCRIPTION
**Pull request by jazmit**

### CODE REVIEW

File: py3status/modules/taskwarrior.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+# import your useful libs here
+from time import time
+import subprocess
+
+class Py3status:
+    cache_timeout = 5
+
+    def taskWarrior(self, i3s_output_list, i3s_config):
+        active_task = subprocess.check_output(
```
Comment (ultrabug) [inline py3status/modules/taskwarrior.py]: can you please try to make use of the `shlex` python module as a good practice ?

File: py3status/modules/taskwarrior.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+"""
+Display currently active (started) taskwarrior tasks.
+
+Configuration parameters:
+    - cache_timeout : how often we refresh this module in seconds
+
+Requires
+    - jq
```
Comment (ultrabug) [inline py3status/modules/taskwarrior.py]: please sort them alphabetically

File: py3status/modules/taskwarrior.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
```
Comment (ultrabug) [inline py3status/modules/taskwarrior.py]: overall note, if you can, please make sure your code passes the `yapf` formatting tool

**Comment by ultrabug** [comment]
Thanks a lot for your contribution @jazmit, I've added a few comments please.

**Comment by jazmit** [comment]
Thanks for the feedback, I ended up removing the dependence on awk and jq in favour of the python json library.

**Comment by ultrabug** [comment]
Sorry for the delay @jazmit, thanks again for your contribution !

**Comment by jazmit** [comment]
No worries happy to help.  Thanks for all the great work with py3status.

**Pull request merged**